test/integration/basic_test.go

```diff
@@ -350,6 +351,66 @@ func TestUEBuffering(t *testing.T) {
 	testUEDetach(t, fillExpected(&tc))
 }
 
+func TestSliceMeter(t *testing.T) {
+	setup(t, ConfigDefault)
+	defer teardown(t)
+
+	testCases := []testCase{
+		{
+			sliceConfig: &pfcpiface.NetworkSlice{
+				SliceName: "P4-UPF-1",
+				SliceQos: pfcpiface.SliceQos{
+					UplinkMbr:    20000,
+					UlBurstBytes: 10000,
+					DownlinkMbr:  10000,
+					DlBurstBytes: 10000,
+					BitrateUnit:  "Kbps",
+				},
+			},
+			expected: p4RtValues{
+				sliceMeter: &sliceMeter{
+					sliceID: 1,
+					TC:      3,
+					rate:    20000000,
+					burst:   10000,
+				},
+			},
+			desc: "Uplink rate higher",
+		},
+		{
+			sliceConfig: &pfcpiface.NetworkSlice{
```

Comment:
nit: do you see a value in having two test cases here? Since it's desirable for unit tests, the goal of an integration test is just to check if the flow (from northbound to UP4) is correct and the datapath entries are properly installed.

Reply:
Yes, I agree, this should be tested as part of unit tests. Since we don't have unit tests for that part of the code, I would prefer to leave the test as is, and remove when we'll add unit tests.